spyrograph/epitrochoid/epicycloid.py

```diff
@@ -9,10 +9,18 @@
 from spyrograph._cycloid import _Cycloid
 
 class Epicycloid(_Cycloid, Epitrochoid):
-    """Model of a epicycloid which is a special case of a epitrochoid where the
-    circle is rolling around the outside of the fixed circle and has 1/3 the
-    radius of the fixed circle
-    """
+    """A class that represents an epicycloid, which is a curve traced by a point on a circle
+    that rolls around the outside of a fixed circle. This is a special case of an epitrochoid
+    where the rolling circle has a radius that is one-third the radius of the fixed circle.
```

Comment:
Please remove the second sentence as "one-third the radius of the fixed circle." is not relevant to this particular example